pkg/controller/subjectpermission/subjectpermission_controller.go

```diff
@@ -132,54 +131,36 @@ func (r *ReconcileSubjectPermission) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, nil
 	}
 
-	if len(instance.Spec.ClusterPermissions) != 0 {
-		// build a clusterRoleBindingNameList which consists of clusterRoleName-subjectName
-		crClusterRoleBindingNameList := buildClusterRoleBindingCRList(instance)
-
-		// check ClusterRoleBindingName
-		populateCrClusterRoleBindingNameList := populateClusterRoleBindingNames(crClusterRoleBindingNameList, clusterRoleBindingList)
-
-		var CreatedClusterRoleBindingCount int
-		// loop through crClusterRoleBindingNameList
-		for _, clusterRoleBindingName := range populateCrClusterRoleBindingNameList {
-			// get the clusterRoleName by spliting the clusterRoleBindng name
-			clusterRBName := strings.Split(clusterRoleBindingName, "-")
-			clusterRoleName := clusterRBName[0]
-			subjectName := clusterRBName[1]
-
-			// create a new clusterRoleBinding on cluster
-			newCRB := newClusterRoleBinding(clusterRoleName, subjectName, instance.Spec.SubjectKind)
-			err := r.client.Create(context.TODO(), newCRB)
-			if err != nil {
-				if k8serr.IsAlreadyExists(err) {
-					continue
-				}
-
-				var clusterRoleNames []string
-				clusterRoleNames = append(clusterRoleNames, clusterRoleName)
-
-				reqLogger.Error(err, "Failed to create clusterRoleBinding")
+	// for every ClusterPermission
+	var CreatedClusterRoleBindingCount int
```

Comment:
Any specific reason to capitalize these?

Reply:
no reason :) not sure why I did... will change it